tensorboard/webapp/metrics/views/main_view/common_selectors_test.ts

```diff
@@ -736,7 +734,11 @@ describe('common selectors', () => {
     });
 
     it('returns empty list when no experiments are provided', () => {
-      expect(selectors.factories.getRenderableRuns([])(state)).toEqual([]);
+      state.app_routing!.activeRoute = {
```

Comment:
The name of the test doesn't seem quite right any longer. Maybe change to say "returns empty list when route does not contain experiments".